hbase-replication/src/main/java/org/apache/hadoop/hbase/replication/ReplicationPeers.java

```diff
@@ -166,8 +175,8 @@ public void onConfigurationChange(Configuration conf) {
     for (ReplicationPeerImpl peer : peerCache.values()) {
       try {
         peer.onConfigurationChange(
-          ReplicationUtils.getPeerClusterConfiguration(peer.getPeerConfig(), conf));
-      } catch (ReplicationException e) {
+          ReplicationPeerConfigUtil.getPeerClusterConfiguration(conf, peer.getPeerConfig()));
+      } catch (IOException e) {
         LOG.warn("failed to reload configuration for peer {}", peer.getId(), e);
```

Comment:
Not your code, but, should this failure be fatal?

Reply:
For me I think a warn or even an error is enough, this just means one peer fails to load new configuration, it is not necessary to kill the region server?

Reply:
Yes true, the RS should not abort.